docs/pages/cli.mdx

```diff
@@ -11,6 +11,13 @@ xSuite utility provides the following commands:
     - `--no-git` - skip git initialization
 - `xsuite build` - builds a contract; currently it has the following options:
     - `-r, --recursive` - builds all contracts under the directory
+- `xsuite build-reproducible` - builds a contract in a reproducible way
+    - `--dir <DIR>` - specify the directory, which contains one or multiple smart contracts to be built
+    - `--image <IMAGE>` - specify the image, that is used to build the contract (e.g. multiversx/sdk-rust-contract-builder:v6.1.1)
+- `xsuite verify-reproducible` - verifies a contract, that has been built in a reproducible way
```

Comment:
Correct. You need to have a reproducible build ready somewhere in order to verify it. It will send everything (code, hashes, metadata) to the endpoint and then verify it and publish it to the explorer for you.